css/styles.css

```diff
@@ -0,0 +1,314 @@
+/*reset*/
+/* 
+*::before,
+*::after,
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Roboto', sans-serif;
+} */
```

Comment:
`REQUIRED`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.